code/modules/examine/examine.dm

```diff
@@ -49,8 +49,17 @@
 	description_holders["icon"] = "[bicon(A)]"
 	description_holders["desc"] = A.desc
 
+// Byond seemingly calls stat, each tick.
+// Calling things each tick can get expensive real quick.
+// So we slow this down a little.
+// See: http://www.byond.com/docs/ref/info.html#/client/proc/Stat
 /client/Stat()
 	. = ..()
+	if(holder)
```

Comment:
I contemplated making this `is_admin`, but I figured that would be more overhead than just checking for `holder`, not to mention an additional proc-call.